sympy/utilities/lambdify.py

```diff
@@ -178,6 +178,102 @@ def _import(module, reload=False):
 # linecache.
 _lambdify_generated_counter = 1
 
+def pre_treatment_cse(args_f, expr):
+    r"""
+    This function does a pretreatment of the expression to prevent errors when
+    putting a Derivative as an argument to lambdify. The Derivatives present in
+    the arguments that also appear in the function are replaced by various strings.
+    This function returns the new expressions with the instances of Derivatives replaced
+    and a dictionnary that contains the associations of changes
+    (ex : Derivative(x, u) : "x0")
+    """
```

Comment:
Please follow the docstring style guide: https://docs.sympy.org/latest/contributing/docstring.html